apps/synapse-portal-framework/src/components/elportal/ELSupportedByNIH.tsx

```diff
@@ -65,19 +37,23 @@ const ELSupportedByNIH: React.FC = () => {
             backgroundRepeat: 'no-repeat',
             backgroundSize: 'contain',
           }}
-        ></Box>
+        />
         <Typography
           variant="body1"
           sx={{
             pt: '32px',
+            fontSize: '18px',
           }}
         >
           The National Institute on Aging (NIA) is a leading research
           organization dedicated understanding the nature of aging, and extend
           the healthy, active years of life. As part of the National Institutes
           of Health (NIH), NIA supports cutting-edge research on aging and
           age-related diseases, including Alzheimer's disease and other forms of
-          dementia. With a mission to improve the health and well-being of older
+          dementia.
+          <br />
+          <br />
```

Comment:
Probably just use two `Typography` elements instead of adding `br`, since these will correspond to `p` tags, which would be appropriate for these two paragraphs.